test/integration/http_protocol_integration.h

```diff
@@ -42,11 +42,18 @@ class HttpProtocolIntegrationTest : public testing::TestWithParam<HttpProtocolTe
   //
   // Upstream and downstream protocols may be changed via the input vectors.
   // Address combinations are propagated from TestEnvironment::getIpVersionsForTest()
-  static std::vector<HttpProtocolTestParams> getProtocolTestParams(
-      const std::vector<Http::CodecType>& downstream_protocols = {Http::CodecType::HTTP1,
-                                                                  Http::CodecType::HTTP2},
-      const std::vector<Http::CodecType>& upstream_protocols = {Http::CodecType::HTTP1,
-                                                                Http::CodecType::HTTP2});
+  static std::vector<HttpProtocolTestParams>
+  getProtocolTestParams(const std::vector<Http::CodecType>& downstream_protocols =
```

Comment:
Let's specifically call out in the PR description that this PR is enabling HTTP/3 by default in the integration tests.

Reply:
Done